v3/src/components/graph/adornments/movable-value/movable-value-adornment-model.ts

```diff
@@ -59,7 +59,7 @@ export const MovableValueModel = AdornmentModel
     }
   }))
   .actions(self => ({
-    addValue(aValue?: number) {
+    addValue(aValue?: number, init=false) {
```

Comment:
Unused argument `init`?

Reply:
Yeah, no longer needed. I've removed it.